examples/recipes/entity-attribute-value/cube.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extendContext: (req) => {
+    const requestValues = {
+      token: req.headers.authorization,
+      host: req.headers.host,
+      path: req.url.split('?')[0]
+    };
+
+    // console.log(requestValues);
```

Comment:
Remove unused comment?

Reply:
No, thank you! I deliberately left it for the recipe user's convenience 😄